torchvision/prototype/transforms/_color.py

```diff
@@ -6,11 +6,47 @@
 
 from torchvision.prototype import datapoints
 from torchvision.prototype.transforms import functional as F, Transform
+from typing_extensions import Literal
```

Comment:
We should be consistent here. Sometimes you are using `: int` over `Literal[1, 3]`. I'm not going to oppose the removal the removal, but I personally would keep it given that it is going to become a language feature really soon: https://github.com/pytorch/vision/pull/7110#discussion_r1080876196

Reply:
Literal isn't supported by torchscript.  I had it everywhere, but had to remove it later on. LMK if you prefer `int` vs `Literal` for the non-JIT cases

Reply:
No, let's strife for consistency.